nbextensions/bulkImportCell/testAppObj-prod.js

```diff
@@ -0,0 +1,34 @@
+define(['./jobsData', 'common/jobs', 'json!./testAppObj-prod.json'], (
```

Comment:
this mimics what `testAppObj-ci.js` does, but with the data coming from a different file